scripts/docker_container.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+COMMAND="${1:-jupyter}"
+SRC_DIR=${SRC_DIR:-`pwd`}
+CONTAINER_NAME=${CONTAINER_NAME:-pymc}
+PORT=${PORT:-8888}
+
+# stop and remove previous instances of the pymc container to avoid naming conflicts
+if [[ $(docker ps -aq -f name=${CONTAINER_NAME}) ]]; then
+   echo "Shutting down and removing previous instance of ${CONTAINER_NAME} container..."
+   docker rm -f ${CONTAINER_NAME}
+fi
+
+# COMMAND can be either `build` or `bash` or `jupyter`
+if [[ $COMMAND = 'build' ]]; then
+   docker build \
+      -t ${CONTAINER_NAME} \
+      -f $SRC_DIR/scripts/Dockerfile $SRC_DIR
+
+elif [[ $COMMAND = 'bash' ]]; then
+   docker run -it -v $SRC_DIR:/home/jovyan/workspace --rm  --name ${CONTAINER_NAME} ${CONTAINER_NAME} bash
+else
```

Comment:
Might be nice to have an explicit elif for the `jupyter` option, not necessary though.  I also think it'd be nice to keep the bit at the end that [launches the notebook in the browser automatically](https://github.com/pymc-devs/pymc/blob/66215ace64b73f7c4ff2cc431ced9154e33da5e3/scripts/start_container.sh#L30-L32) but that's just a "nice to have" and could be a later PR.

Reply:
Hi yes, the $COMMAND is set to `jupyter` as default in `COMMAND="${1:-jupyter}"`. So, after the docker image is built, we can run with `jupyter` as a default option.